front/cypress/e2e/admin/projects/project_description_settings.cy.ts

```diff
@@ -0,0 +1,44 @@
+import { randomString } from '../../../support/commands';
+
+describe('Project description', () => {
+  let projectId: string | null = null;
+
+  beforeEach(() => {
+    cy.setAdminLoginCookie();
+    if (projectId) {
+      cy.apiRemoveProject(projectId);
+    }
```

Comment:
There's a chance I will still change this bit, but feel free to already review the rest of the PR.